### ISSUE TITLE
Disable VSYNC

### ISSUE THREAD

**Issue by teseo-sw** [rockchip-linux/kernel#25]
Neither eglSwapInterval(0) nor vblank_mode=0 works as expected. Is there any other way to disable VSYNC?

**Comment by wzyy2** [comment]
Why do you want to disable vsync? It's unnecessary, if you want to test performance, you can just swap the offscreen buffers.

There is no way to disable VSYNC. Both swap a buffer to the front layer via gles api(x11/wayland) and show them via drm api(gbm) will wait VSYNC.

**Comment by teseo-sw** [comment]
Yes, I can measure performance by rendering offscreen or putting monitor asleep.
What if I need those additional 10 frames VSYNC rips me off?

**Comment by wzyy2** [comment]
libmali-fbdev could avoid DRM force vsync.
